systemtest/src/test/java/io/strimzi/systemtest/upgrade/StrimziUpgradeST.java

```diff
@@ -172,22 +180,24 @@ void testChainUpgrade() throws Exception {
         }
     }
 
+    // TODO: remove disabled tag after fix of issue #3685
     @Test
+    @Disabled("Disabled because of bug https://github.com/strimzi/strimzi-kafka-operator/issues/3685")
     void testUpgradeKafkaWithoutVersion() throws IOException {
         File dir = FileUtils.downloadAndUnzip(latestReleasedOperator);
+        File kafkaPersistent = new File(dir, "strimzi-" + latestReleasedVersion + "/examples/kafka/kafka-persistent.yaml");
+        coDir = new File(dir, "strimzi-" + latestReleasedVersion + "/install/cluster-operator/");
+        kafkaVersions = FileUtils.downloadYaml("https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/" + latestReleasedVersion + "/kafka-versions.yaml");
```

Comment:
Why does this needs to be a class parameter? Why not pass it directly into the single method using it?